src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -325,6 +327,30 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
         });
     }
 
+    private async updateProjectTagsFromAsset(asset: IAssetMetadata) {
```

Comment:
I would like to see a new unit tests validating this behavior in the editor page.

Reply:
Integrated the test about forcing the tag from the asset metadata directly into the existing unit tests with minimal impact on mocks.  Otherwise I had to duplicate lot of mock and beforeall code with very few changes.